doc/next-node-rules.md

```diff
@@ -73,5 +73,5 @@ Occurs if the `next_node` blocks returns a value that isn't defined as a questio
 next_node do
   outcome :red
 end
-outcome :green
+outcome :green?
```

Comment:
Good catch. I've changed it to `outcome :blue` so that it's less confusing. Fix-up commit: 87af37eab369ce184941969f39f3911b95083957.